source/_components/media_player.enigma2.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: "Volumio Media Player"
+description: "How to set up the Enigma2 media player platform"
+date: 2017-04-07
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Media Player
+ha_release: 0.42
+---
+
+The `Enigma2` platform allows you to control [Enigma2](http://wiki.blue-panel.com/index.php/Enigma2_(en)) linux based settopboxes through web API
+from Home Assistant.
+
+To test if your web api works: Navigate your browser at http://192.168.1.x/web/powerstate, you should see there an XML output.
+x - put ip address of your box
+
+To add Enigma2 player to your installation, add the following to
+your `configuration.yaml` file.
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: enigma
+    host: 192.168.1.x
+    port: 80
+    username: root
+    password: pass
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.